components/label.json

```diff
@@ -31,6 +31,11 @@
       "#normal": {
         "text": "@theme-text-error-normal"
       }
+    },
+    "commonWhite": {
```

Comment:
I agree... can you add the 8 common text tokens (https://www.figma.com/file/yYfvFEEr3UkW1yUKkW7zEH/Tokens---Theme-Color?node-id=3078%3A6338) to the label component?
```
"label":{
      "common":{
           "white": "@theme-common-text-white",
           "gray": "@theme-common-text-gray",
           ....
      }
}
```

Reply:
Pushed commit with all the common tokens.
[Added all common styles to label](https://github.com/momentum-design/tokens/pull/214/commits/199da43615b7e12a1ac60120520539bd8444be0a)
Changing this conversation to resolved.